server/tests/conftest.py

```diff
@@ -1,7 +1,10 @@
 import pytest
-
+import os
 from text_generation_server.pb import generate_pb2
 
+os.environ["USE_PREFIX_CACHING"] = "1"
+os.environ["ATTENTION"] = "flashinfer"
```

Comment:
?

Reply:
This is for the server tests, those are run in isolation (without the router/launcher).

Therefore we need those variables to be set (those variables are enforced to exist in the python code, in order to minize issues in the resolution of their values.